sciLifeLab_utils/run_delivery.py

```diff
@@ -6,31 +6,19 @@
 
 def main(arg):
     project = os.path.split(os.path.realpath(arg.source))[1]
-    move_from_path = "{}/*/results/".format(arg.source)
-    pathway = glob.glob(move_from_path)
-    samples = []
-    for path in pathway:
-        try:
-            samples.append(os.path.basename( os.path.dirname(path.rstrip('/'))))
-
-        except IndexError:
-            pass
-    
-    
-    for sample in samples:
-        dest = "/proj/{}/INBOX/{}/QC_analysis/{}".format(arg.uppnexid,project,sample)
-        try:
-             os.mkdir("/proj/{}/INBOX/{}/QC_analysis".format(arg.uppnexid,project))
-          
-        except OSError:
-            pass
-        try:
-           os.mkdir(dest)
-       
-        except OSError:
-            pass
-       
-        cmd = ["rsync", "-auhvr", "{}/{}/results".format(arg.source, sample), dest]
+    move_from_path = "{}/*/".format(arg.source)
+    samples_path = glob.glob(move_from_path)
+    #create in inbox folder for project
+    #dest = "/proj/{}/INBOX/{}/".format(arg.uppnexid,project)
+    dest = "/Users/vezzi/Desktop/"
```

Comment:
it requires some improvement... but for now I prefer to go for a quick and dirty solution